nomad/grapl-core.nomad

```diff
@@ -275,21 +290,13 @@ variable "user_session_table" {
   description = "What is the name of the DynamoDB user session table?"
 }
 
-variable "sysmon_generator_queue" {
-  type = string
-}
+# variable "osquery_generator_queue" {
+#   type = string
+# }
 
-variable "sysmon_generator_dead_letter_queue" {
-  type = string
-}
-
-variable "osquery_generator_queue" {
-  type = string
-}
-
-variable "osquery_generator_dead_letter_queue" {
-  type = string
-}
+# variable "osquery_generator_dead_letter_queue" {
+#   type = string
+# }
```

Comment:
Should this be deleted? Otherwise an inline comment as to why it's here would be helpful.

Reply:
Thanks I forgot about this

Reply:
deleted this commented out code (and some final remaining traces of osquery-generator) and wrote a story to bring back osquery-generator and generic-subgraph-generator once we've got a plugin system to put them in, and the pipeline is migrated: https://app.zenhub.com/workspaces/grapl-6036cbd36bacff000ef314f2/issues/grapl-security/issue-tracker/934